pkg/model/dev.go

```diff
@@ -546,13 +546,29 @@ func (dev *Dev) SetDefaults() error {
 	return nil
 }
 
-func (b *BuildInfo) setBuildDefaults() {
+func (b *BuildInfo) setBuildDefaults() error {
 	if b.Context == "" {
 		b.Context = "."
 	}
-	if _, err := url.ParseRequestURI(b.Context); err != nil && b.Dockerfile == "" {
```

Comment:
@pchico83 @jLopezbarb In which cases the context could be a URI? Isn't needed anymore?

Reply:
Right now we could have a GitHub repo as context and it was building the image, don't know if this change has break this scenario. @mnevadom could you check it pls?

Reply:
If we need to recover that check, lets include a comment in the code to be aware of that please and thanks for the explanation @jLopezbarb !

Reply:
Agree. Ok